packages/connection-encrypter-plaintext/src/index.ts

```diff
@@ -91,7 +91,7 @@ class Plaintext implements ConnectionEncrypter {
         id: localId.toBytes(),
         pubkey: {
           Type: type,
-          Data: localId.publicKey ?? new Uint8Array(0)
+          Data: localId.publicKey === null ? new Uint8Array(0) : (PublicKey.decode(localId.publicKey).Data ?? new Uint8Array(0))
```

Comment:
To fix the build errors, I think this needs to be something like:

```suggestion
          Data: localId.publicKey == null ? new Uint8Array(0) : (PublicKey.decode(localId.publicKey).Data ?? new Uint8Array(0))
```

`x == null` is true if `x` is `null` or `undefined`, `x === null` is only true if `x` is `null`.